demo/meeting/src/types/index.ts

```diff
@@ -34,3 +34,17 @@ export type ContentShareControlContextType = {
   toggleContentShare: () => Promise<void>;
   togglePauseContentShare: () => void;
 };
+
+export enum DeviceLabels {
```

Comment:
nit: Cant we reuse the enum from the library?

Reply:
Yes we can. I forgot to export this enum from the library. Deleted now.